bin/rucio

```diff
@@ -1702,19 +2343,41 @@ def list_datasets_rse(args):
 
     """
     client = get_client(args)
-    if args.long:
-        table = []
-        for dsn in client.list_datasets_per_rse(args.rse):
-            table.append(['%s:%s' % (dsn['scope'], dsn['name']), '%s/%s' % (str(dsn['available_length']), str(dsn['length'])), '%s/%s' % (str(dsn['available_bytes']), str(dsn['bytes']))])
-        table.sort()
-        print(tabulate(table, tablefmt=tablefmt, headers=['DID', 'LOCAL FILES/TOTAL FILES', 'LOCAL BYTES/TOTAL BYTES']))
+    if cli_config == 'rich':
+        status = Status('Fetching datasets at RSE', spinner=CLITheme.SPINNER, spinner_style=CLITheme.SPINNER_STYLE, console=console)
+        with status:
+            if args.long:
+                table_data = []
+                for dsn in client.list_datasets_per_rse(args.rse):
+                    table_data.append(['%s:%s' % (dsn['scope'], dsn['name']),
```

Comment:
(This comment applies for all tables using %s formatting) Is there a reason for this not to be f-strings? Those are preferred for readability purposes.

Reply:
No particular reason,  I reused the approach from the original code base.